portfolio.js

```diff
@@ -23,3 +23,169 @@ menuitems.forEach((item) => {
+ + 'Lorem ipsum dolor sit amet consectetur adipisicing elit.Lorem ipsum dolor sit amet consectetur adipisicing elit.'
+  + 'Odit exercitationem quidem eius illum',
+
+  tech: [' ', 'html', 'Ruby on rails', 'css', 'javaScript'],
+  link: '#',
+  psource: '#',
+};
+
+const body = document.querySelector('body');
+const open = document.querySelectorAll('.see-more');
+const open2 = document.querySelectorAll('.see-more2');
+const open3 = document.querySelectorAll('.see-more3');
+const open4 = document.querySelectorAll('.see-more4');
+function openfunction() {
+  const main = document.createElement('section');
+  main.className = 'main';
+  const popup = document.createElement('div');
+  popup.className = 'popup';
+  popup.innerHTML = "<div class='upper'>"
+  + "<h2 class='stronger' id='head'>Header </h2>"
+  + "<button type='button' class='close'><i class='fa fa-times'></i></button></div>"
+  + "<div id='mylist' class='canopy'>OK</div>"
+  + "<div class='mycard'>Nothing</div>"
+  + "<div class='footer'>"
+  + "<p class='des'>Lorem ipsum dolor sit amet consectetur adipisicing elit."
+  + ' Odit exercitationem quidem eius illum.</p> '
+  + "<div class='last-foot'><div id='mytech' class='language'>OK</div>"
+  + "<div class='bgroup'>"
+  + "<a class='see-live id='link' href='#'><button type='button' class='live '>See live<img src='images/live.svg' alt=''></button></a>"
+  + "<a class='see-live' id='res' href='#'><button type='button' class='live'>See source<img src='images/git.png' alt=''></button></a></div></div></div>";
+  // popup.append(list);
+  main.append(popup);
+  body.append(main);
+
+  const close = document.querySelector('.close');
+  close.addEventListener('click', () => {
+    body.removeChild(main);
+  });
+}
+
+function pe1() {
+  openfunction();
+  document.getElementById('head').innerHTML = p1.hname;
+  // pnamelist
+  const list = p1.pname.join('</li class="noitem"><li>');
+  document.getElementById('mylist').innerHTML = list;
+  const techlist = p1.tech.join('</li><li>');
+  document.getElementById('mytech').innerHTML = techlist;
+  // imagelist
+  document.querySelector('.mycard').innerHTML = p1.img;
+
+  document.querySelector('.des').innerHTML = p1.description;
+  document.querySelector('.res').querySelector.setAttribute('href', p1.psource);
+  document.querySelector('.link').querySelector.setAttribute('href', p1.link);
+}
+
+function pe2() {
+  openfunction();
+  document.getElementById('head').innerHTML = p1.hname;
+  // pnamelist
+  const list = p2.pname.join('</li class="noitem"><li>');
+  document.getElementById('mylist').innerHTML = list;
+  const techlist = p2.tech.join('</li><li>');
+  document.getElementById('mytech').innerHTML = techlist;
+  // imagelist
+  document.querySelector('.mycard').innerHTML = p2.img;
+  document.querySelector('.des').innerHTML = p2.description;
+  document.querySelector('.res').querySelector.setAttribute('href', p2.psource);
```

Comment:
- [ ] As pointed out above, you do not need to create multiple functions if you have the projects as a collection(array of objects)